zerver/tornado/event_queue.py

```diff
@@ -1579,6 +1604,8 @@ def process_notification(notice: Mapping[str, Any]) -> None:
         process_custom_profile_fields_event(event, cast(list[int], users))
     elif event["type"] == "realm_user" and event["op"] == "add":
         process_realm_user_add_event(event, cast(list[int], users))
+    elif event["type"] == "user_group" and event["op"] == "update" and "name" in event["data"]:
+        process_user_group_name_update_event(event, cast(list[int], users))
```

Comment:
Why is the check `"name" in event["data"]`?

Reply:
Ahh, I see this in the API changelog; definitely worth a comment on this detail, either here or in the `process_user_group_name_update_event` function.